hawkular-inventory-parent/hawkular-inventory-service/src/main/java/org/hawkular/inventory/handlers/InventoryHandlers.java

```diff
@@ -53,8 +58,9 @@
 public class InventoryHandlers {
 
     private static final MsgLogger log = InventoryLoggers.getLogger(InventoryHandlers.class);
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
Suggestion (minor), JsonUtil.getMapper() has also same object and it is used in other handlers..